PodcastGenerator/core/feed_generator.php

```diff
@@ -18,7 +18,7 @@ function generateRSS()
     // Set the feed header with relevant podcast informations
```

Comment:
I think if we have season and episode numbers provided we don't need to use file modification timestamps for the order of episodes. With the current approach it could be possible to have an episode of season 2 to be newer than one of season 3 for example. We need to adjust this in the feed generator before a possible merge

Reply:
The modification timestamp should still be used to determine whether or not the episode is published yet, right? Otherwise we open up a whole other can of worms.

Reply:
The other issue is that we need to check the XML sidecar file for every episode in order to sort them by episode and season number. While I agree that we should sort the feed this way instead of by timestamp, I think it will need to be a different body of work.

Reply:
> The modification timestamp should still be used to determine whether or not the episode is published yet, right? Otherwise we open up a whole other can of worms.

Yes and that's the problem I see with this PR though. While I think it's necessary to have season and episodes support we need it to be more robust and user friendly somehow.